misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss

```diff
@@ -159,4 +159,4 @@
       }
     }
   }
-}
+}
```

Comment:
```suggestion
}
```

Reply:
Sorry that i forgot the last line. Hope you can merge it now and i didnt forget anything else.

Reply:
you need to accept/commit the suggestion ;)

Reply:
can´t accept or commit!